src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/repository/OffenderRepository.kt

```diff
@@ -2,7 +2,9 @@ package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.repository
 
 import org.springframework.data.jpa.repository.JpaRepository
 import org.springframework.stereotype.Repository
-import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.entity.Assessment
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.entity.Offender
 
 @Repository
-interface AssessmentRepository : JpaRepository<Assessment, Long>
+interface OffenderRepository : JpaRepository<Offender, Long> {
+  fun findByPrisonerNumber(prisonerNumber: String): Offender?
```

Comment:
Think it should return list of Offenders?

Reply:
At the moment we are only going to have one entry for each prisonerNumber so it will only return one at the moment.